backend/src/data/migrations/20220818201316_courses_add_original_id.ts

```diff
@@ -0,0 +1,21 @@
+import { Knex } from 'knex';
+
+enum ColumnName {
+  ORIGINAL_ID = 'original_id',
+}
+
+const TABLE_NAME = 'courses';
+
+async function up(knex: Knex): Promise<void> {
+  return knex.schema.table(TABLE_NAME, (table) => {
+    table.string(ColumnName.ORIGINAL_ID).nullable();
```

Comment:
Let's provide unique constraint for `<original_id, vendor_id>` pair to restrict duplicates on db level